AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DecalAdded.py

```diff
@@ -164,51 +159,56 @@ def AtomEditorComponents_Decal_AddedToEntity():
         get_opacity_property = decal_component.get_component_property_value(AtomComponentProperties.decal('Opacity'))
         Report.result(Tests.opacity_property_set, get_opacity_property == 0.5)
 
-        # 8. Set Sort Key property on Decal component
+        # 8. Set Normal Map Opacity property on Decal component
+        decal_component.set_component_property_value(AtomComponentProperties.decal('Normal Map Opacity'), value=0.5)
+        get_opacity_property = decal_component.get_component_property_value(AtomComponentProperties.decal('Normal Map Opacity'))
```

Comment:
nit: get_opacity_property is from the test in step 7. reusing it here could possibly result in unexpected outcomes if the value isn't correctly overwritten in this assignment.
consider renaming this variable for debugging clarity

Reply:
fixed!